resolwe_bio/processes/differential_expression/deseq.yml

```diff
@@ -23,6 +23,12 @@
       type: list:data:expression
       description: >
         Control samples (replicates)
+    - name: filter
+      label: Filtering of raw counts
```

Comment:
Write what is the parameter, not what is does. Think of this as along parameter name. For example, "Raw counts threshold" or "Raw counts filtering threshold".